app/workflows/lists/conversion/sections/after-opening.yml

```diff
@@ -9,12 +9,11 @@ tasks:
     actions:
       - title: Email the regional delivery officer
 
-  - slug: redact-and-send-funding-agreement-documents
-    title: Redact and send funding agreement documents
+  - slug: redact-and-send-documents-to-be-published-on-gov-uk
+    title: Redact and send documents to be published on GOV.UK
     hint: |
```

Comment:
We have some wayward pipes `|` that could be removed while we are here

Reply:
Just to refresh my brain the Pipe means something like "output this as literal HTML" so the markdown becomes html?

Reply:
It means to preserve newlines.

If we do this (no pipe):
```yaml
hint:
  List:
  * thing
  * thing
```

We will get the following output (there won't be any formatting because the new line is necessary):
```
List: *thing *thing
```

So, we need the pipe when we want spaces between paragraphs, or to use bullet points or numbered lists.